src/commands/utility/remindme.js

```diff
@@ -0,0 +1,328 @@
+const {
+  SlashCommandBuilder,
+  EmbedBuilder,
+  ActionRowBuilder,
+  ButtonBuilder,
+  ButtonStyle,
+  ModalBuilder,
+  TextInputBuilder,
+  TextInputStyle,
+} = require('discord.js');
+
+const {
+  parseDate,
+  cancelReminder,
+  replyToInteraction,
+  disableReminderButtons,
+  updateCountdownEmbed,
+  resetReminder,
+  formatReminderDate,
+} = require('../../utils/remindme-functions');
+const { translateLanguage } = require('../../languages/index');
+
+const activeReminders = new Map();
+let reminderCounter = 0;
+
+module.exports = {
+  data: new SlashCommandBuilder()
+    .setName('remindme')
+    .setDescription(translateLanguage('remindme.commandDescription'))
+    .addStringOption((option) =>
+      option
+        .setName('time')
+        .setDescription(translateLanguage('remindme.timeOptionDescription'))
+        .setRequired(true)
+    )
+    .addStringOption((option) =>
+      option
+        .setName('message')
+        .setDescription(translateLanguage('remindme.messageOptionDescription'))
+        .setRequired(true)
+    ),
+
+  async execute(interaction) {
+    const timeInput = interaction.options.getString('time');
+    const message = interaction.options.getString('message');
+    const reminderDate = parseDate(timeInput);
+
+    if (!reminderDate) {
+      return replyToInteraction(
+        interaction,
+        translateLanguage('remindme.invalidFormat'),
+        true
+      );
+    }
+
+    const timeDiff = reminderDate.getTime() - Date.now();
+    if (timeDiff <= 0) {
+      return replyToInteraction(
+        interaction,
+        translateLanguage('remindme.timeInFuture'),
+        true
+      );
+    }
+
+    // Se crea un nuevo recordatorio sin cancelar otros ya existentes
+    await this.startReminder(interaction, reminderDate, message, timeDiff);
+  },
+
+  async startReminder(interaction, reminderDate, message, originalDuration) {
```

Comment:
Please be more specific with "originalDuration" name, for example if it is the time in ms counting from now, it could be called something like: "timeInMsFromNow" or "timeUntilReminderMs"